pandas/core/groupby/groupby.py

```diff
@@ -2506,6 +2513,10 @@ def _get_cythonized_result(
             key = base.OutputKey(label=name, position=idx)
             output[key] = result
 
+        # error_msg is "" on an frame/series with no rows or columns
```

Comment:
why is this even needed?

Reply:
For `resample.test_base.test_resample_empty_dataframe`. This test explicitly expects the result to be an empty dataframe when the input it empty as well.